insight_testsuite/temp/src/sessionization.py

```diff
@@ -0,0 +1,36 @@
+# stdlib
```

Comment:
would be nice to have brief comments in the header explaining what this file does